nats-server/configuration/README.md

```diff
@@ -134,10 +134,10 @@ authorization: {
 
 | Property | Description | Default |
 | :--- | :--- | :--- |
-| `max_connections` | Maximum number of active client connections. | `64K` |
-| `max_control_line` | Maximum length of a protocol line \(including combined length of subject and queue group\). Increasing this value may require [client changes](../../developing-with-nats/connecting/misc.md#set-the-maximum-control-line-size) to be used. Applies to all traffic. | `4Kb` |
-| `max_payload` | Maximum number of bytes in a message payload. Reducing this size may force you to implement [chunking](../../developing-with-nats/connecting/misc.md#get-the-maximum-payload-size) in your clients. Applies to client and leafnode payloads. It is not recommended to use values over 8Mb but `max_payload` can be set up to 64Mb. | `1Mb` |
-| `max_pending` | Maximum number of bytes buffered for a connection Applies to client connections. | `64Mb` |
+| `max_connections` | Maximum number of active client connections. | `65536` |
+| `max_control_line` | Maximum length of a protocol line \(including combined length of subject and queue group\). Increasing this value may require [client changes](../../developing-with-nats/connecting/misc.md#set-the-maximum-control-line-size) to be used. Applies to all traffic. | `4KiB` |
```

Comment:
No, it is base 10, so maybe you are right that Kb is ambiguous, but then it is 4KB (4 * 1024 * 1024 = 4194304 bytes), not 4Kib.

Reply:
Base 10 would be 4 * 1000 * 1000. This is base-2, which is why you're multiplying by 1024.

From Wikipedia, here is a unit chart: 
![image](https://user-images.githubusercontent.com/726063/133283925-26fde35c-5c48-41d3-b79c-a908b956d229.png)

`MiB` is the one that uses 1024.

Reply:
just based on SI definitions, lowercase `mb` would mean "milli-bit`, which is not a unit anyone should ever use.

Reply:
Ok, you got me.. But from your same link you see the Memory column that says MB... so that's the one we want.

Reply:
In other words, we want to use the "Memory" column units, which is KB, MB, GB, TB. This is how most of the time memory usage is reported.

Reply:
It's _accepted_ to do MB in memory related contexts, but it is still ambiguous... MiB is explicit that it is powers of 2, and can't be confused for powers of 10.

Regardless, this isn't my docs, so... you can do what you want. I was just trying to help alleviate future confusion for consumers of the docs.

Reply:
Appreciate the comment and trying to make the doc better, but I just think that MB is more commonly used (and understood) that MiB. Not sure what machine/OS you are running, but I have a mac, if I check "About" and check storage or memory, it uses unit as MB/GB, etc.. and I am sure others do too, not specific to Apple.

You are certainly correct, but to be frank, I am sure there would be more people confused by MiB than MB...

Reply:
Yep, definitely pros and cons to both approaches. The ideal solution would be to allow the user to hover over the unit and see some alt-text that says this is base-2 or binary or something, but that's probably not easy with git book.